packages/components/popover/src/Popover.test.tsx

```diff
@@ -44,20 +44,20 @@ describe('Popover', () => {
       )
 
       // ...by default, only trigger elements are rendered
-      expect(document.querySelector('[data-spark-component="popover-anchor"]')).toBeInTheDocument()
-      expect(document.querySelector('[data-spark-component="popover-trigger"]')).toBeInTheDocument()
+      expect(document.querySelector('[data-spark-component="Popover.Anchor"]')).toBeInTheDocument()
```

Comment:
I opened a thread to discuss this point on Slack using this PR :) I worked on the Chrome extension rencently and think it was weird to show "popover-anchor" instead of the displayName of the component inside the extension. I'll edit back to kebab-case.